tests/import_test.go

```diff
@@ -965,5 +965,12 @@ var _ = Describe("Preallocation", func() {
 		Entry("Blank image", true, func() *cdiv1.DataVolume {
 			return utils.NewDataVolumeForBlankRawImage("import-dv", "100Mi")
 		}),
+		Entry("Blank block DataVolume", true, func() *cdiv1.DataVolume {
+			if !f.IsBlockVolumeStorageClassAvailable() {
+				Skip("Storage Class for block volume is not available")
+			}
+
+			return utils.NewDataVolumeForBlankRawImageBlock("import-dv", "100Mi", f.BlockSCName)
```

Comment:
A full alloc should be pretty slow, so there should be enough time to verify that a pod actually was created.

Reply:
The default 270 seconds should be enough for anybod^W writing 100Mi of zeros.

Reply:
Well the new part here is that for a block device we DO create a pod if preallocation is set to true. So our test should actually verify the pod is created.

Reply:
The test verifies that the preallocation annotation has been created, and it's only created when the pod started AND finished successfully.

Reply:
Gotcha, didn't realize the annotation was created only if the pod started and finished.